aws/resource_aws_macie2_invitation_test.go

```diff
@@ -0,0 +1,175 @@
+package aws
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/macie2"
+	"github.com/hashicorp/aws-sdk-go-base/tfawserr"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	"github.com/terraform-providers/terraform-provider-aws/aws/internal/envvar"
+)
+
+const (
+	EnvVarMacie2MemberEmail             = "AWS_MACIE2_MEMBER_EMAIL"
```

Comment:
We need two environment variables here, one for Invitation Accepter with the email address of the main account, and one for Invitation and Member with the email address of the alternate account. I suggest

* `AWS_MACIE2_ACCOUNT_EMAIL`
* `AWS_MACIE2_ALTERNATE_ACCOUNT_EMAIL`